exonum-java-binding-common/src/test/java/com/exonum/binding/common/serialization/json/StoredConfigurationGsonSerializerTest.java

```diff
@@ -51,6 +81,17 @@ void nullPointerException() {
         () -> StoredConfigurationGsonSerializer.fromJson(jsonConfiguration));
   }
 
+  @Test
+  void readConfiguration() {
```

Comment:
This test is not very useful as it has too relaxed expectations.

A test of decoding a reference configuration *might be* useful because we can either unit- or system-test this code. However, to be useful, it must assert that the decoded configuration is equal to the expected (or that it contains the expected values).

I'd suggest 
1. Adding a focused test just for PublicKey
2. Improving this test to make meaningful assertions (or removing it)

Reply:
Sounds like a great improvement, thanks !